documentation/src/docs/metrics.md

```diff
@@ -477,36 +416,36 @@ controller.stream.truncated_latency_ms
 ```
 controller.stream.created.Counter
 controller.stream.create_failed_global.Counter
```

Comment:
In some places the metrics end with `.Counter` and in others they don't. Please, let's keep a consistent way of referring to the metrics names.

Reply:
fixed.